### ISSUE TITLE
Latest Chirons shipping with 1.3.0 firmware. Source please

### ISSUE THREAD

**Issue by aenertia** [ANYCUBIC-3D/ANYCUBIC_CHIRON#4]
Hi there.

I recieved my chiron with 1.3.0 ; which isn't listed even on the support site.

Would be helpful to have sources updated with latest fixes.

**Comment by Sv3nergie** [comment]
It seems software support is zero here, I've asked before and they don't update source-code. They don't understand that opensource and software support is everything to be succesful in the 3d-printing world.

If you search around , you'll find a fork that supports the original lcd from the I3 mega. I've compiled it for my modded direct drive Chiron and it's zooming around at 90ms/sec very silently on 5 tmc2130 stepper drivers. The only problem is the auto bed leveling lcd support on the Chiron is not on the I3 Mega and you loose this function on the LCD. You can still autolevel and probe with Gcode commands in Pronterface though. So not that bad. Just not that clean.

Marlin 1.1.9 is stable as hell and you'll never need an update again. I don't understand why Anycubic uses an old version of Marlin.

Greetz

**Comment by jsprinceiii** [comment]
AC did not commit 1.3.0 to this repository :(

However.. You can find it here..

https://github.com/ANYCUBIC-3D/ANYCUBIC_CHIRON_V1.3.0

**Comment by Sv3nergie** [comment]
yes, I missed it too at first... not much changed though. Some Sd card read fixes, and slower extruder feed rate it seems.

**Comment by jsprinceiii** [comment]
Yes, several minor changes.

**Comment by aenertia** [comment]
The first mod I want to make is to expose the secondary uart pins to be
usable for ESP8266 wifi bridge.

This isn't supported on the old marlin tree, and requires a 2.x branch ;
it's needed if you want to add wifi AND retain the screen/usb port.

**Comment by Sv3nergie** [comment]
you can try to compile derhopp's 2.x branch of the I3 Mega. It failed to compile with a z-offset lcd error on my part. If you don't use the probe it should work. His 1.1.9 branch works fine with some minor changes in the firmware settings for the Chiron except the lcd auto leveling part but that can be done with gcode commands in pronterface. Wish Anycubic would release a 2.x update, they have the knowledge of their coded lcd. Greetz.

**Comment by mikehoogstraat** [comment]
So we cannot change the output to the display ? I do not see any of the code delaing with their manual  levleling process
